store/sqlstore/channel_store_experimental.go

```diff
@@ -684,16 +683,14 @@ func (s SqlChannelStoreExperimental) buildLIKEClause(term string) (likeClause, l
 	return
 }
 
-func (s SqlChannelStoreExperimental) buildFulltextClause(term string) (fulltextClause, fulltextTerm string) {
+func (s SqlChannelStoreExperimental) buildFulltextClause(term string, searchColumns string) (fulltextClause, fulltextTerm string) {
 	if !s.IsExperimentalPublicChannelsMaterializationEnabled() {
-		return s.SqlChannelStore.buildFulltextClause(term)
+		return s.SqlChannelStore.buildFulltextClause(term, "c.Name, c.DisplayName, c.Purpose")
```

Comment:
Should we be passing the `searchColumns` parameter up here instead of overriding it?

Reply:
Yes, that makes sense

Reply:
Fixed :)